packages/host/scripts/patch-hermes.rb

```diff
@@ -4,13 +4,20 @@
   raise "React Native Node-API cannot reliably patch JSI when React Native Core is prebuilt."
 end
 
-VENDORED_HERMES_DIR ||= `npx react-native-node-api vendor-hermes --silent '#{Pod::Config.instance.installation_root}'`.strip
-if Dir.exist?(VENDORED_HERMES_DIR)
-  Pod::UI.info "Hermes vendored into #{VENDORED_HERMES_DIR.inspect}"
-else
-  raise "Hermes patching failed. Please check the output above for errors."
+if ENV['REACT_NATIVE_OVERRIDE_HERMES_DIR'].nil?
+  VENDORED_HERMES_DIR ||= `npx react-native-node-api vendor-hermes --silent '#{Pod::Config.instance.installation_root}'`.strip
```

Comment:
Is there any reason for this to be `||=` rather than `=`? As `VENDORED_HERMES_DIR` is a local variable, I think it can only be unset at this point?

```suggestion
  VENDORED_HERMES_DIR = `npx react-native-node-api vendor-hermes --silent '#{Pod::Config.instance.installation_root}'`.strip
```

Reply:
It's my understanding that Ruby scripts aren't caching requires the same way Node.js would.

I had this originally because it prevents re-execution of the command if the .rb file is (transitively) "required" multiple times.